docker-compose.services.yml

```diff
@@ -50,7 +50,8 @@ services:
   es:
     build:
       context: docker/opensearch
-    restart: 'always'
+      dockerfile: 'opensearch.Dockerfile'
+    restart: always
```

Comment:
hm it's weird that you need to add this - `dockerfile` is an optional arg for `build` and I think that since Dockerfile is the default image that docker is searching in the directory specified in `context` it should work (it worked for all the intel users and me on m1). But if it doesn't break anything and makes ES work for you is ok, but then instead of adding a new dockerfile rename the old one :)